db.yml

```diff
@@ -8997,972 +8754,1281 @@ modules:
+          sceUltQueueDestroy: 0xF37862DE
+          sceUltQueuePop: 0x1AD58A53
+          sceUltQueuePush: 0xA7E78FF9
+          sceUltQueueTryPop: 0x2A1A8EA6
+          sceUltQueueTryPush: 0x6D356B29
+          sceUltReaderWriterLockDestroy: 0xB1FEB79B
+          sceUltReaderWriterLockLockRead: 0x9AD07630
+          sceUltReaderWriterLockLockWrite: 0xF5F63E2C
+          sceUltReaderWriterLockTryLockRead: 0x2629C055
+          sceUltReaderWriterLockTryLockWrite: 0x944FB222
+          sceUltReaderWriterLockUnlockRead: 0x218D4743
+          sceUltReaderWriterLockUnlockWrite: 0x2A5741F5
+          sceUltSemaphoreAcquire: 0xF220D3AE
+          sceUltSemaphoreDestroy: 0x8EC57420
+          sceUltSemaphoreRelease: 0x65376E2D
+          sceUltSemaphoreTryAcquire: 0xAF15606D
+          sceUltUlthreadExit: 0x1E401DF8
+          sceUltUlthreadGetSelf: 0xA798C5D7
+          sceUltUlthreadJoin: 0x63483381
+          sceUltUlthreadRuntimeDestroy: 0x4E9A745C
+          sceUltUlthreadRuntimeGetWorkAreaSize: 0x5435C586
+          sceUltUlthreadTryJoin: 0xB4CF88AC
+          sceUltUlthreadYield: 0xCAD57BAD
+          sceUltWaitingQueueResourcePoolDestroy: 0xC9E96714
+          sceUltWaitingQueueResourcePoolGetWorkAreaSize: 0x644DA029
+  SceUsbd:
+    nid: 0x3525FE7A
     libraries:
-      SceSblSmCommForKernel:
-        kernel: true
-        nid: 0xCD3C89B6
+      SceUsbdForUser:
+        kernel: false
+        nid: 0xC3AEAB67
         functions:
-          ksceSblSmCommCallFunc: 0xDB9FC204
-          ksceSblSmCommStartSmFromData: 0x039C73B1
-          ksceSblSmCommStartSmFromFile: 0x7863A0CC
-          ksceSblSmCommStopSm: 0x0631F8ED
-  SceMsif:
-    nid: 0x3305A0FE
-    libraries:
-      SceMsifForDriver:
+          sceUsbdAttach: 0xEACEAE86
+          sceUsbdAttachCompositeLdd: 0xCA8F2F59
+          sceUsbdClosePipe: 0xAA3AF5D5
+          sceUsbdEnd: 0x5736A150
+          sceUsbdGetDescriptor: 0xAE671F22
+          sceUsbdGetDescriptorSize: 0xB357AC81
+          sceUsbdGetDeviceAddress: 0xC54F9D11
+          sceUsbdGetDeviceList: 0x2DE0239E
+          sceUsbdGetDeviceSpeed: 0x6C5AACD0
+          sceUsbdGetIsochTransferStatus: 0xA0A2C826
+          sceUsbdGetTransferStatus: 0xCDF5B6E0
+          sceUsbdInit: 0x77D58B31
+          sceUsbdIsochTransferData: 0x594D82FD
+          sceUsbdOpenDefaultPipe: 0x4A26DDCC
+          sceUsbdOpenPipe: 0xF94521A6
+          sceUsbdReceiveEvent: 0x16FEE05D
+          sceUsbdRegisterCallback: 0x9A820167
+          sceUsbdRegisterCompositeLdd: 0x2A7C0263
+          sceUsbdRegisterLdd: 0x6AD28166
+          sceUsbdResetDevice: 0xDEB3BE59
+          sceUsbdTransferData: 0x716048C1
+          sceUsbdUnregisterCallback: 0x0585EA80
+          sceUsbdUnregisterLdd: 0x473692CC
+      SceUsbdForDriver:
         kernel: true
-        nid: 0xB706084A
+        nid: 0xA0EBCA41
         functions:
-          ksceMsifDisableSlowMode: 0x75848756
-          ksceMsifEnableSlowMode: 0x4B751CE6
-          ksceMsifGetMsInfo: 0xD0307849
-  
```

Comment:
these are original function before the patch.
so can you change other side?
and maybe give postfix(like 2~4) is better than prefix(`_`)

but if you think it's private function, just remain the comment :)

Reply:
I'm sorry but I can not figure out where to fix it.
Function names such as _sceErrorGetExternalString are defined in SceSysLibTrace.

Reply:
i mentioned c2a2ec7205 commit parts, and you added same name function to other side.
imo, `_` prefix only can have an internal function(private method)
btw technically we can call it if that was defined `nid`. :/

Reply:
No @d3m3vilurr there are _sce prefixed functions that are exported.
The issue is the ksce naming: it would give k_sce.
Moreover sometimes __sce.

Reply:
@CelesteBlue-dev I know. and you told repeat and repeat about objection of the vitasdk's decisions that why we use `ksce` instead `ForDriver/Kernel` and why we don't use the sony's export name.
please stop.

Reply:
so @CelesteBlue-dev you can suggest another name, and if that name has really good, we can change every name.
but it is also sdk, so full changing wouldn't be acceptable :)